client/src/pages/LessonHome/LessonHome.js

```diff
@@ -90,18 +95,59 @@ const LessonHome = ({ navigation }) => {
             }
 
             // Need to fetch image uri
-            // eslint-disable-next-line no-await-in-loop
-            const uri = await trackPromise(
-              downloadImageFile(
-                currentCourseId,
-                currentUnitId,
-                currentLessonId,
-                item._id,
-                fileType,
-              ),
-            )
-
-            formattedItem.imageURI = uri
+            /* eslint-disable no-loop-func */
```

Comment:
I think you might need to write a comment justifying these no-loop-func eslint disables

Reply:
Alternatively you can talk to Ashay about maybe disabling these lint catches